src/App.jsx

```diff
@@ -1,12 +1,77 @@
-import './App.css';
+import axios from 'axios'
+import React, {useState} from 'react';
+
 
 function App() {
+    const [countries, setCountries] = useState([]);
+    const [search,setSearch] = useState('');
+    const [filteredCountries, setFilteredCountries] = useState([]);
+    const fetchCountries = async () => {
+        try {
+            const response = await axios.get("https://restcountries.com/v3.1/all");
+            const sortedCountries = response.data
+                .map((country) => ({
+                    name: country?.name.common || "Onbekend",
+                    population: country?.population || 0,
+                    flags: country?.flags.png || "",
+                    region: country?.region || "Onbekend",
+
+                }))
+                .sort((a, b) => a.population - b.population);
```

Comment:
Wees ervan bewust dat je met de sort methode de originele array aanpast. Vaak is het handig om een kloon te maken van de originele array om vervolgens op de gekloonde array de methodes op los te gooien. Zo blijft de originele array intact.

hoe kloon je een array? zo dus:

`const clonedArray = [...originalArray]`